src/main/java/org/matsim/run/modules/SnzBerlinScenario25pct2020JR.java

```diff
@@ -0,0 +1,287 @@
+/*-
+ * #%L
+ * MATSim Episim
+ * %%
+ * Copyright (C) 2020 matsim-org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.matsim.run.modules;
+
+import com.google.inject.Provides;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.episim.EpisimConfigGroup;
+import org.matsim.episim.EpisimUtils;
+import org.matsim.episim.TracingConfigGroup;
+import org.matsim.episim.model.FaceMask;
+import org.matsim.episim.model.Transition;
+import org.matsim.episim.model.input.ActivityParticipation;
+import org.matsim.episim.model.input.CreateDistrictRestrictionsFromCSVJR;
+import org.matsim.episim.policy.FixedPolicy;
+import org.matsim.episim.policy.FixedPolicy.ConfigBuilder;
+import org.matsim.episim.policy.Restriction;
+
+import javax.inject.Singleton;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Snz scenario for Berlin.
+ *
+ * @see AbstractSnzScenario
+ */
+public final class SnzBerlinScenario25pct2020JR extends AbstractSnzScenario2020 {
```

Comment:
Do you need a copy of the whole file? It is possible to extend/delegate to the existing scenario and only make additional changes?